deploy_website_to_surge.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+if [type = pull_request && ${WEBSITE_CHANGED}]
+then
+  npx surge --project ./build --domain wix-yoshi-${TRAVIS_PULL_REQUEST}.surge.sh;
+
+  #inspired by https://damien.pobel.fr/post/github-api-from-travisci/
+  CURL_RESULT = curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST \
+    -d "{\"body\": \"You could check new website at http://wix-yoshi-${TRAVIS_PULL_REQUEST}.surge.sh/yoshi 😉\"}" \
```

Comment:
```suggestion
    -d "{\"body\": \"You could check new website at https://wix-yoshi-${TRAVIS_PULL_REQUEST}.surge.sh/yoshi\"}" \
```

Also I think that you can use html tags to style it a bit 😉

Reply:
Well, it's so opinionated...
Could you please provide some info how you would like to see the message?

Reply:
That's correct, let's start with the message provided and improve later on down the road if we'll feel that's important enough.